ros2_ws/src/vision/path_detection/path_detection/obstacles.py

```diff
@@ -57,11 +55,12 @@ def __init__(self):
         self.state = STATE.LINE_FOLLOWING
 
         # lidar parameters
-        self.declare_parameter("/LIDARTrimMin", 1.31)
-        self.declare_parameter("/LIDARTrimMax", 4.97)
-        self.declare_parameter("/ObstacleFOV", math.pi/6)
-        self.declare_parameter("/ObstacleDetectDistance", 1.5)  # meters
-        self.declare_parameter("/FollowingDirection", 1)
+        self.declare_parameter("/LIDARTrimMin", 2.8808)         # radians = 165.0575 degrees
+        self.declare_parameter("/LIDARTrimMax", 0.2576)         # radians = 14.7 degrees
+
```

Comment:
probably should remove this blank line so that the lidar parameters are together